.github/workflows/pr-labels.yml

```diff
@@ -28,9 +28,25 @@ jobs:
         with:
           script: |
             const [owner, repo] = '${{ github.repository }}'.split('/');
-            github.rest.issues.addLabels({
+            const pLabels = github.rest.issues.listLabelsOnIssue({
               owner,
               repo,
               issue_number: context.issue.number,
               labels: ${{ env.LABELS }}
```

Comment:
should you still give the param `labels` to retrieve the list ?

Reply:
It's not possible to have params/input to a PR triggered workflow but I can move it out to `env` to be more easily configurable